integration_test/sql/subquery.exs

```diff
@@ -18,6 +18,7 @@ defmodule Ecto.Integration.SubQueryTest do
            TestRepo.all(from p in subquery(query), select: p)
   end
 
+  @tag :map_boolean_in_subquery
```

Comment:
Sorry but I don' understand exactly what this tag is about. Can you please explain why this does not work on sqlite_ecto2?

Usually the tags are about the database features that we don't support but this is unfortunately phrased in terms of Ecto features.